app/controllers/comment.js

```diff
@@ -87,24 +99,26 @@ module.exports = {
       // CHECKS IF THE USER HAS ALREADY UPVOTED THE COMMENT
       comment.votes.upVotes.user.filter(user => {
         if (JSON.stringify(user) === JSON.stringify(userId)) {
-          return res.status(HttpStatus.BAD_REQUEST).json({ error: 'Bad request' })
+          return res.status(HttpStatus.BAD_REQUEST).json({
+            error: 'Bad request'
+          })
         }
       })
       // CHECKS IF THE USER HAS ALREADY DOWNVOTED THE COMMENT
```

Comment:
we can remove this entire logic as we dont have downvote in our platform.

Reply:
@devesh-verma I talked to @jaskirat2000 He told me that we will downvote on comments but not on posts.